actionManifest-webcraft.json

```diff
@@ -0,0 +1,62 @@
+{
+    "tag": "quiz-teams-app",
+    "id": "com.microsoft.sample.quiz.mike06072005",
```

Comment:
i think this id should be generic like
com.microsoft.sampleapp.quiz

you can verify what poll app has and on similar lines you can add id here

Reply:
Okay I have removed this file